### ISSUE TITLE
Last chapter and details of fanfiction are missing

### ISSUE THREAD

**Issue by xypha** [FicHub/fichub.net#3]
Last chapter is always missing when I try to download an epub of a fanfiction
[Example 1](https://www.fanfiction.net/s/13268034/1/A-Cell-of-a-Good-Time) - epub included only 4 chapters instead of 5 (at the time of the download request - 15.Apr.2021, 1.08 am IST) - 5th chapter was missing
[Example 2](https://www.fanfiction.net/s/11575324/1/Silver) - epub included only 25 chapters instead of 26 (at the time of the download request - 15.Apr.2021, 1.24 am IST) - 26th chapter was missing

The missing chapters were available when I checked on Google Chrome.

Also, missing fiction details on the first page of the epub when making an epub from fanfiction.net -
Type of fiction - `Crossover` (example 1)
Fandom - `Dragon Ball Z & Young Justice` (example 1)
Fiction rating - `Rated: M` (example 1)
Language - `English` (example 1)
Genre - `Sci-Fi/Humor` (example 1)
Pairing/Major characters - `[L. Ren, Nora V.] Jaune A., Pyrrha N.`  (example 2)

adding these details would help - especially when one stops/completes reading the epub and comes back to the story at a later date

creating this issue because I am not on discord

Thank you for making FicHub

**Comment by iridescent-beacon** [comment]
I've kicked the updater to refresh the two specific instances you linked. The [help section on the homepage](https://fichub.net/#help) tries to explain the underlying issue. Please let me know if you have wording suggestions to make it clearer.

There's a long standing todo item about extending the metadata that is included in the generated files. I'll update this issue when that is addressed. I'll also see about putting the todo list somewhere visible to people who don't used discord.

Thank you! I'm glad it's been useful :)

**Comment by iridescent-beacon** [comment]
Enough groundwork has been laid out that I've deployed an initial update covering parts of this. For FFN: rating, language, chars/pairing, genre, review count, favorite count, and follow count have been added if present. When it gets long it wraps less-than-ideally which needs fixed. Fandoms, fandom category, and whether it's a crossover are still pending. Additional metadata for non-FFN sources is pending.

A scrub is back-filling existing exports, until that finishes only new exports will include the extra metadata. Example 1 & 2 from the OP have updated and will include the above fields when exported.
